src/panels/config/cloud/login/cloud-login.ts

```diff
@@ -214,19 +218,39 @@ export class CloudLogin extends LitElement {
     this._requestInProgress = true;
 
     try {
+      let pipelines: AssistPipeline[] = [];
+      let preferred_pipeline: string | null = null;
+      try {
+        const response = await listAssistPipelines(this.hass);
```

Comment:
let's not await here, but after login

Reply:
We need to fetch pipelines before to know if it's new or not.

Reply:
we can start the fetch, but don't have to await it

Reply:
```ts
const listPromise = listAssistPipelines(this.hass);
await login();
const response = await listPromise;
```

Reply:
Not sure about concurrent requests 😅 Listing pipelines should be faster than login + pipeline creation.

Reply:
it is faster, so no need to wait before starting the login, as we don't need it before the login is finished 🙃